python/lsst/ts/cRIOpy/M1M3/BoosterValveWidget.py

```diff
@@ -0,0 +1,193 @@
+
+        self.__followingErrorChart = TimeChart(
+            {
+                "Following Error (N)": [
+                    "Primary Max",
+                    "Primary Min",
+                    "Secondary Max",
+                    "Secondary Min",
+                ]
+            }
+        )
+
+        layout.addWidget(TimeChartView(self.__followingErrorChart), 1, 0, 1, 2)
+
+        self.setLayout(layout)
+
+        m1m3.forceActuatorData.connect(self._forceActuatorData)
+
+    @Slot(map)
+    def _forceActuatorData(self, data):
+        primaryFEMax = np.max(data.primaryCylinderFollowingError)
+        primaryFEMin = np.min(data.primaryCylinderFollowingError)
+
+        secondaryFEMax = np.max(data.secondaryCylinderFollowingError)
+        secondaryFEMin = np.min(data.secondaryCylinderFollowingError)
+
+        self.__followingErrorChart.append(
+            data.timestamp, [primaryFEMax, primaryFEMin, secondaryFEMax, secondaryFEMin]
+        )
+
+        self._pmax.setValue(primaryFEMax)
+        self._pmin.setValue(primaryFEMin)
+        self._smax.setValue(secondaryFEMax)
+        self._smin.setValue(secondaryFEMin)
+
+
+class Accelerometer(QWidget):
+    """
+    Display DC accelerometer values relevant for booster valve triggering.
+
+    Parameters
+    ----------
+    m1m3 : `SALComm`
+        M1M3 SAL remote.
+    """
+
+    def __init__(self, m1m3):
+        super().__init__()
+        layout = QGridLayout()
+
+        operationalLayout = QFormLayout()
+
+        operationalLayout.addRow(
+            "Accelerometer Trigger Enabled",
+            OnOffLabel(m1m3.boosterValveSettings, "accelerometerTriggerEnabled"),
+        )
+        for axis in ["X", "Y", "Z"]:
+            operationalLayout.addRow(
+                f"Open {axis}",
+                DegS2(m1m3.boosterValveSettings, f"accelerometer{axis}TriggerOpen"),
+            )
+            operationalLayout.addRow(
+                f"Close {axis}",
+                DegS2(m1m3.boosterValveSettings, f"accelerometer{axis}TriggerClose"),
+            )
+        operationalLayout.addRow(
+            "Accelerometer Triggered",
+            OnOffLabel(m1m3.boosterValveStatus, "accelerometerTriggered"),
+        )
+
+        layout.addLayout(operationalLayout, 0, 0)
+
+        plotAxis = SALAxis(
+            "Angular Acceleration (deg/s<sup>2</sup>)", m1m3.accelerometerData
+        )
+
+        dataLayout = QFormLayout()
+
+        for axis in ["X", "Y", "Z"]:
+            dataLayout.addRow(
+                f"Angular {axis}",
+                DegS2(m1m3.accelerometerData, f"angularAcceleration{axis}"),
+            )
+            plotAxis.addValue(f"{axis}", f"angularAcceleration{axis}")
+
+        layout.addLayout(dataLayout, 0, 1)
+
+        layout.addWidget(SALChartWidget(plotAxis), 1, 0, 1, 2)
+
+        self.setLayout(layout)
+
+        m1m3.boosterValveSettings.connect(self._boosterValveSettings)
+
+    def _boosterValveSettings(self, data):
+        print(data)
```

Comment:
Is this function just to print the data on screen?

Reply:
I though I force commit a push that removed that, apparently wasn't the case. Fixed.